DavidGonzalezZumaya/Collections/src/ArrayList/ArrayListResult.java

```diff
@@ -0,0 +1,40 @@
+package ArrayList;
+
+import java.util.List;
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class ArrayListResult {
+    final static String EMPTY = "ERROR!";
+
+    public static void main(String[] args) {
+
+        Scanner in = new Scanner(System.in);
+
+        int numberOfLines = in.nextInt();
+
+        List<List<Integer>> matrix = new ArrayList<>();
```

Comment:
It is recommended that the name of the collections and maps start with singular name and them the name of the interface, example: numberList